cw_decred/lib/wallet_creation_credentials.dart

```diff
@@ -14,10 +14,12 @@ class DecredRestoreWalletFromSeedCredentials extends WalletCredentials {
   final String mnemonic;
 }
 
+
 class DecredRestoreWalletFromWIFCredentials extends WalletCredentials {
   DecredRestoreWalletFromWIFCredentials(
       {required String name, required String password, required this.wif, WalletInfo? walletInfo})
-      : super(name: name, password: password, walletInfo: walletInfo);
+      : assert(false), // Not feasable with current decred implementation. TODO: Surely this is not the correct way to do this.
+      super(name: name, password: password, walletInfo: walletInfo);
```

Comment:
Isn't this for watch-only wallets, where the wif is the xpub? Don't know though. Especially since `assert` only works in dev mode, not in prod, I agree there would need to be a better way to disable this feature if decred doesn't support it. Can probably just throw an exception inside the constructor.